system/index.js

```diff
@@ -29,18 +29,21 @@ const userFn = getFunction(USER_FUNCTION_URI);
 module.exports = async ({headers, payload}) => {
   if (DEBUG) {
     console.log('==System Function Start==');
-    console.log(`HEADERS: ${JSON.stringify(headers)}`);
+    console.log(`HEADERS: ${headers}`);
     console.log(`ORIGINAL PAYLOAD: ${payload}`);
     console.log(`MIDDLEWARE_FUNCTION_URI: ${MIDDLEWARE_FUNCTION_URI}`);
     console.log('==Middleware Function(s) Start==');
   }
 
+  const req = { payload };
```

Comment:
One goal of doing this was to ensure the headers became available to the middleware. I'm not sure the `headers` here is what we really want as I don't love the `Headers` object defined in riff as the interface. It leaks too much of riff into the middleware chain.

```suggestion
  const req = { payload, headers };
```